provider/azure_private_dns.go

```diff
@@ -0,0 +1,419 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
is this the right copyright?

Reply:
@njuettner, people used that also in other PR's, which have recently been merged.
Can you answer?

Reply:
Correct, please don't drop it we need it.